src/pkg/cli/client/config.go

```diff
@@ -0,0 +1,5 @@
+package client
+
+type Config struct {
```

Comment:
I am a bit less inclined to name this config, config should be something modifiable by the user for a different behaviour, but here, it is more like a hidden state, and end up being saved in the state folder. If we add user configurable properties here later, it would be a mix of both.

Reply:
agree; renamed to state